src/asm2wasm.h

```diff
@@ -1331,7 +1331,8 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
 
   // apply memory growth, if relevant
   if (preprocessor.memoryGrowth) {
-    emscripten::generateMemoryGrowthFunction(wasm);
+    EmscriptenGlueGenerator generator(wasm, Address(0));
```

Comment:
not a big deal, but the `Address(0)` looks weird here, since we don't mean "0" (I think?). If it had a default value, we could avoid writing it here?

Reply:
Yes, this makes even more sense later on when I want to use `Address(0)` (or something) to signal "no stack pointer specified, stack pointer is a global, use that instead". Although I also need an entirely separate way to specify *which* global is the stack pointer so I'd need a separate constructor anyway...

But still yeah, asm2wasm shouldn't have to care what `Address(0)` may or may not mean. (Also, following @dschuff's prior suggestion to emit grow memory via the JS API we won't need this in asm2wasm at all, but future PRs are future PRs.)